src/ProjectService.Data.Provider.MsSql.Ef/ProjectServiceDbContext.cs

```diff
@@ -12,6 +12,9 @@ public class ProjectServiceDbContext : DbContext, IDataProvider
         public DbSet<DbProject> Projects { get; set; }
         public DbSet<DbProjectFile> ProjectsFiles { get; set; }
         public DbSet<DbProjectUser> ProjectsUsers { get; set; }
+        public DbSet<DbTask> Task { get; set; }
```

Comment:
use names with 's' on the end, Tasks and TaskProperties

Reply:
done